pyfstat/make_sfts.py

```diff
@@ -811,13 +808,16 @@ def run_makefakedata(self):
         check_ok = self.check_cached_data_okay_to_use(cl_mfd)
         if check_ok is False:
             helper_functions.run_commandline(cl_mfd)
-            if np.all([os.path.isfile(f) for f in self.sftfilepath.split(";")]):
-                logging.info(f"Successfully wrote SFTs to: {self.sftfilepath}")
-            else:
-                raise RuntimeError(
+            if not np.all([os.path.isfile(f) for f in self.sftfilenames]):
+                raise IOError(
                     f"It seems we successfully ran {mfd},"
                     f" but did not get the expected SFT file path(s): {self.sftfilepath}."
                 )
+            logging.info(f"Successfully wrote SFTs to: {self.sftfilepath}")
+            if np.any([lalpulsar.ValidateSFTFile(sft) for sft in self.sftfilenames]):
+                raise IOError(
+                    f"Some of the output SFTs '{self.sftfilepath}' failed validation."
```

Comment:
```suggestion
                    f"Some of the output SFTs '{self.sftfilepath}' failed validation."
                    " by lalpulsar.ValidateSFTFile."
```

Reply:
Actually it was never raising that message, just printing the XLALError and quitting. New version, first saying what we're doing and then the internal failure:
```
19:16 INFO    : Successfully wrote SFTs to: TestData/H-4_H1_1800SFT_TestWriter-700000000-7200.sft;TestData/L-4_L1_1800SFT_TestWriter-700000000-7200.sft
19:16 INFO    : Now validating each SFT file...
Unable to open TestData/H-4_H1_1800SFT_TestWriter-700000000-7200.sftx : No such file or directory
XLAL Error - XLALValidateSFTFile (SFTutils.c:2208): SFT validation error code 1 on file TestData/H-4_H1_1800SFT_TestWriter-700000000-7200.sftx.
XLAL Error - XLALValidateSFTFile (SFTutils.c:2208): Internal function call failed
```